README.md

```diff
@@ -63,7 +63,7 @@ const { lastIssues } = await graphql(`query lastIssues($owner: String!, $repo: S
 })
 ```
 
-Set default options
+Create a new client and set defaults for it. Defaults are not global for every client.
```

Comment:
It's correct that defaults are not set for every client, but they are inherited. 

Maybe it would be clear to say:

> Create a new client with custom defaults

🤔

Reply:
The main point of confusion, for me, that I wanted to clear up was that `.defaults` could be used without modifying the settings of other clients. Perhaps I'm not seeing how inheritance is used here?

For example, in the following code, all 3 clients have different authorizations right?

```javascript
const octokitGraphQl = require("@octokit/graphql");

const graphql1 = octokitGraphQl.defaults({
  headers: {
    authorization: `token secret123`,
  },
});

const graphql2 = octokitGraphQl;

const graphql3 = octokitGraphQl.defaults({
  headers: {
    authorization: `token foobar`,
  },
});


```

How can we change the README to communicate that subsequent new clients will not share configuration state with previous ones? If there is a case where configuration state is shared between clients, we should communicate that too.

Reply:
> For example, in the following code, all 3 clients have different authorizations right?

Yes, but when you do sth like this

```js
const graphql1 = octokitGraphQl.defaults({
  headers: {
    authorization: `token secret123`,
  },
});

const graphql2 = graphql1.defaults({
  headers: {
    user-agent: 'my-user-agent/v1.2.3'
  }
});
```

then `graphql2` "inherits" the authorization header defaultf from `graphql1`

Reply:
Ah, I see thanks. Looking at the code I see this is just a light wrapper around `@octokit/request` which describes that in its README. I think that being verbose would help people, like me, who come to this project without having used `@octokit/request` before. I've updated the PR with additional examples. Let me know what you think.